### PR TITLE
Interface

### PR DESCRIPTION
**Pull request by ijeomaodinko**
These are react files, 
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- interface for the calculator.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,7 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```
Comment (Nemwel-Boniface) [inline src/App.js]: - [ ] Kindly note that according to the project requirements, you are required to use Class based components in all of your components. This as well should be changed from stateless to stateful components. 

image
 

Below I have included for you how it is supposed to be:

```
// eslint-disable-next-line react/prefer-stateless-function
class App extends Component {
  render() {
    return (<Calculator />);
  }
}
```

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @ijeomaodinko            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an amazing job with your Math magicians so far :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ijeomaodinko** [comment]
fix the App file

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
```
Comment (ShahierNashaat) [inline src/components/Calculator.js:1]: - [ ] Kindly be noticed that the `Calculator` component should import the `calculate.js` file from the logic directory.
```javascript
import React, { Component } from 'react';
import calculate from ../logic;
```

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+          <button type="button" className="one">1</button>
+          <button type="button" className="two">2</button>
+          <button type="button" className="three">3</button>
+          <button type="button" className="add  last">+</button>
+          <br />
+
+          <button type="button" className="zero">0</button>
+          <button type="button" className="point">.</button>
+          <button type="button" className="equalto  last">=</button>
```
Comment (ShahierNashaat) [inline src/components/Calculator.js:45]: - [ ] Kindly be noticed that to implement all calculator operations you should implement the click event for all buttons.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @ijeomaodinko, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
- [ ] Kindly be noticed that you should create a `logic` directory that should have an `operate.js` file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js) and a `calculate.js` file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+// import './index.css';
```
Comment (1GORDON) [inline src/index.js:3]: Great job @ijeomaodinko 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is not regarded as a good practice.

File: src/App.js [hunk 5]
```diff
@@ -1,29 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+// import React from 'react';
```
Comment (1GORDON) [inline src/App.js:1]: Great job @ijeomaodinko 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is not regarded as a good practice.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @ijeomaodinko ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**